Makefile

```diff
@@ -76,3 +79,8 @@ label: $(GOBIN)/labeler
 mkdocs-serve:
 	docker build -t $(MKDOCS_IMAGE) -f docs/build/Dockerfile docs/build
 	docker run --name mkdocs-serve --rm -v $(PWD):/docs -p $(MKDOCS_PORT):8000 $(MKDOCS_IMAGE)
+
+_docker_compose:
+	@command -v docker-compose > /dev/null || \
+	(curl -sL https://github.com/docker/compose/releases/download/1.29.0/docker-compose-Linux-x86_64 -o /usr/local/bin/docker-compose && \
```

Comment:
It depends on the developer's environment. I think the docker options are simple enough in our case. Can we do it with the `docker` command instead of `docker-composer`?

Reply:
In the long term, when using docker becomes more complex, `docker-compose` is really useful. 
But sure, will change

Reply:
Yes, we can move to docker-compose when the options will be complex.